packages/ui/src/components/va-accordion/VaAccordion.stories.ts

```diff
@@ -1,12 +1,167 @@
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+export const DefaultWithCustomHeader = (args: Args) => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return VaAccordionSetup
+  },
+  template: `
+    <va-accordion v-model="value" style="width: 400px;">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+      >
+        <template #header="{ value }">
+          {{ value ? 'opened' : 'closed' }}
+        </template>
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+export const Multiple = (args: Args) => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return VaAccordionSetup
+  },
+  template: `
+    <va-accordion v-model="valueArray" multiple style="width: 400px;">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+export const Inset = (args: Args) => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return VaAccordionSetup
+  },
+  template: `
+    <va-accordion v-model="value" inset style="width: 400px;">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+export const InsetMultiple = (args: Args) => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return VaAccordionSetup
+  },
+  template: `
+    <va-accordion v-model="valueArray" inset multiple style="width: 400px;">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+export const Popout = (args: Args) => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return VaAccordionSetup
+  },
+  template: `
+    <va-accordion v-model="value" popout style="width: 400px;">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+export const PopoutMultiple = (args: Args) => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return VaAccordionSetup
+  },
+  template: `
+    <va-accordion v-model="valueArray" popout multiple style="width: 400px;">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
 })
```

Comment:
We need additional story for `stateful` prop.

All props should be covered by stories. You can get a list of props from docs: https://ui.vuestic.dev/ui-elements/accordion#props.